.github/workflows/apm-integrations.yml

```diff
@@ -152,6 +152,22 @@ jobs:
         with:
           dd_api_key: ${{ secrets.DD_API_KEY }}
 
+  bullmq:
+    runs-on: ubuntu-latest
+    services:
+      redis:
+        image: redis:7.0-alpine
+        ports:
+          - 6379:6379
+    env:
+      PLUGINS: bullmq
+      SERVICES: redis
+    steps:
+      - uses: actions/checkout@08c6903cd8c0fde910a37f88322edcfb5dd907a8 # v5.0.0
+      - uses: ./.github/actions/plugins/test
+        with:
+          dd_api_key: ${{{{ secrets.DD_API_KEY }}}}
```

Comment:
damn son, heard you like curly braces

Reply:
This happened in the NATS PR as well

Reply:
easy fix! And simon's comment made me "lol"